### PR TITLE
fix(wallet)_: Paraswap contract address

### PR DESCRIPTION
**Pull request by smohamedjavid**
### Summary

This PR fixes ParaSwap contract address displayed on the setting approval cap for a token. Since, we use ParaSwap v6.2, the contract address is changed.

|  Before   | After
| --- | --- | 
|   |  | 

### Review notes

Since we get the approval contract address in the swap proposal, we take that as higher precedence over the hardcoded value in the constants file.

### Platforms

- Android
- iOS

### Steps to test

- Open Status
- Navigate to Swap flow
- Set a spending cap for a token
- Verify the contract address for ParaSwap is `0x6a000f20005980200259b80c5102003040001068`

status: ready

### CODE REVIEW

**Comment by pavloburykh** [comment]
@smohamedjavid thanks for the PR. I have been able to check only showing correct contract in app UI (see screenshot below). Swap testing is blocked by https://github.com/status-im/status-mobile/pull/21541 which should be fixed in scope of https://github.com/status-im/status-mobile/pull/21541

photo_2024-11-18 17 15 08


@smohamedjavid I think once this PR is reviewed by devs, we are okay to merge it and then we will proceed with testing swap in scope of #21541

Please, let me know in case there will be any additional changes based on dev review, so I can re-test the PR. Thank you.

**Review by pavloburykh — APPROVED** [review]

**Comment by smohamedjavid** [comment]
Thanks for checking it @pavloburykh 🙏 

There are no changes apart from the ParaSwap contract address displayed in the UI. 

I will merged it once I get a approval from Devs.

**Review by clauxx — APPROVED** [review]

**Review by shivekkhurana — APPROVED** [review]

**Pull request merged**
